final-task/src/views/header/header.ts

```diff
@@ -193,4 +202,14 @@ export default class HeaderView {
       this.buttonContainer.classList.remove('loginned');
     }
   }
+
+  public updateCartCount(value: number): void {
+    const buttonCart = this.buttonContainer.querySelector(
+      '.header__button_cart',
+    ) as HTMLDivElement;
+    const buttonCartQuantity = buttonCart.querySelector(
+      '.header__button_cart_quantity',
+    ) as HTMLSpanElement;
+    buttonCartQuantity.textContent = String(value);
+  }
```

Comment:
I think the cart icon with a zero indicator is a bad idea. You can simply add a check for null and hide the element in a positive case
![image](https://github.com/ksarise/eCommerce-Application/assets/18985111/72a9087e-25dd-4275-8cbd-1600eab48816)